doc/dev_guide/internals/file_formats.rst

```diff
@@ -91,49 +89,48 @@ database performance doesn't degrade even if all requests refer to the same
 key in the same space.
 
 The transaction processor thread communicates with the WAL writer thread using
-asynchronous (yet reliable) messaging; the transaction processor thread, not
-being blocked on WAL tasks, continues to handle requests quickly even at high
+asynchronous (yet reliable) messaging.
+The transaction processor thread, not being blocked on WAL tasks, continues to handle requests quickly even at high
 volumes of disk I/O. A response to a request is sent as soon as it is ready,
 even if there were earlier incomplete requests on the same connection. In
 particular, SELECT performance, even for SELECTs running on a connection packed
 with UPDATEs and DELETEs, remains unaffected by disk load.
 
 The WAL writer employs a number of durability modes, as defined in configuration
-variable :ref:`wal_mode <index-wal_mode>`. It is possible to turn the write-ahead
-log completely off, by setting
-:ref:`wal_mode <cfg_binary_logging_snapshots-wal_mode>` to *none*. Even
-without the write-ahead log it's still possible to take a persistent copy of the
+variable :ref:`wal.mode <configuration_reference_wal_mode>`.
+It is possible to turn the write-ahead log completely off, by setting the ``wal_mode`` option to *none*.
+Even without the write-ahead log it's still possible to take a persistent copy of the
 entire data set with the :ref:`box.snapshot() <box-snapshot>` request.
 
 An .xlog file always contains changes based on the primary key.
 Even if the client requested an update or delete using
-a secondary key, the record in the .xlog file will contain the primary key.
+a secondary key, the record in the .xlog file contains the primary key.
 
-.. _internals-snapshot:
+..  _internals-snapshot:
 
 The snapshot file format
 ------------------------
 
-The format of a snapshot .snap file is nearly the same as the format of a WAL .xlog file.
-However, the snapshot header differs: it contains the instance's global unique identifier
+The format of a snapshot .snap file is similar to the format of a WAL .xlog file, except for the header and content.
```

Comment:
The whole file is a composition of headers and content. This sentence looks confusing.